crates/eth-rpc/src/servers/web3_rpc.rs

```diff
@@ -0,0 +1,27 @@
+use jsonrpsee::core::{async_trait, RpcResult as Result};
+use reth_primitives::{keccak256, Bytes, H256};
+
+use crate::api::web3_api::Web3ApiServer;
+
+/// The RPC module for the implementing Web3 Api { i.e rpc endpoints prefixed with web3_ }
+#[derive(Default)]
+pub struct Web3Rpc {}
+
+impl Web3Rpc {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[async_trait]
+impl Web3ApiServer for Web3Rpc {
+    fn client_version(&self) -> Result<String> {
+        let kakarot_client_version = format!("kakarot_{}", env!("CARGO_PKG_VERSION"));
```

Comment:
This can also be a lazy static, for now I went with this approach, but if you think a lazy static is better, then I can do that as well.

Reply:
maybe a `lazy_static` in `constants` to keep all the constant definition at the same place indeed?

Reply:
Make sense! Changed.